summingbird-online/src/test/scala/com/twitter/summingbird/online/TopologyPlannerLaws.scala

```diff
@@ -154,20 +154,6 @@ object TopologyPlannerLaws extends Properties("Online Dag") {
     }
   }
 
-  property("Prior to a summer the Node should be a FlatMap Node") = forAll { (dag: MemoryDag) =>
```

Comment:
do we have to lose this test? This is still true unless people opt in, right? It is a shame to lose any of these tests since almost all of them are the result of past bugs.